Routing/RestRouteCollection.php

```diff
@@ -76,10 +76,17 @@ public function setDefaultFormat($format)
      */
     public function all()
     {
+        $regex = <<<REGEX
```

Comment:
I usually work on a 15" Notebook and I really hate it to scroll horizontal in PhpStorm, so I decided to write it that way